src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_Clarification.cshtml

```diff
@@ -310,9 +310,7 @@
                                     <tr class="govuk-table__row">
                                         <th scope="row" class="govuk-table__header">@file.Filename</th>
                                         <td class="govuk-table__cell govuk-table__cell--numeric">
-                                            @*<a href="#" class="govuk-link govuk-link--no-visited-state">
-                                                Remove
-                                                </a>*@
+                                            <a class="govuk-link" asp-action="RemoveClarificationFile" asp-controller="RoatpFinancial" asp-route-ApplicationId="@Model.ApplicationId" asp-route-Filename="@file.Filename">Remove</a>
```

Comment:
it's missing CSS class `govuk-link--no-visited-state`

Reply:
fixed